config/sync/core.entity_form_display.node.stanford_news.default.yml

```diff
@@ -113,25 +113,7 @@ content:
         maxlength_js_summary: null
         maxlength_js_label_summary: 'Content limited to @limit characters, remaining: <strong>@remaining</strong>'
         maxlength_js_enforce: false
-      conditional_fields:
-        a1b2c3d4-e5f6-7890-a1b2-c3d4e5f67890:
-          entity_type: node
-          bundle: stanford_news
-          dependee: layout_selection
-          settings:
-            state: '!visible'
-            reset: false
-            condition: value
-            grouping: AND
-            values_set: 1
-            value: ''
-            values: {  }
-            value_form:
-              -
-                target_id: news_spotlight
-            effect: show
-            effect_options: {  }
-            selector: ''
+      conditional_fields: {  }
```

Comment:
@pookmish we chatted about keeping the `body` field as a part of spotlights last week. I dropped the ball on updating [Ian's ticket](https://stanfordits.atlassian.net/browse/D8CORE-8390) so I'm adding that `body` field back.